src/azure-cli/azure/cli/command_modules/network/commands.py

```diff
@@ -694,7 +694,7 @@ def _make_singular(value):
     with self.command_group('network dns', network_dns_reference_sdk, resource_type=ResourceType.MGMT_NETWORK_DNS) as g:
         g.command('list-references', 'get_by_target_resources')
 
-    with self.command_group('network dns zone', network_dns_zone_sdk) as g:
+    with self.command_group('network dns zone', network_dns_zone_sdk, resource_type=ResourceType.MGMT_NETWORK_DNS) as g:
```

Comment:
Maybe it's better to fix this issue in another PR

Reply:
both are ok I think